src/basis-cash/BasisCash.ts

```diff
@@ -578,7 +578,7 @@ export class BasisCash {
     const oldpool = '0x0F8c89d3fB0b502732b338f1dfb3c465Dc856C8e';
     const newpool = '0x2B26239f52420d11420bC0982571BFE091417A7d';
     const balace = await this.mic23crv.balanceOf(this.myAccount);
-    const gas = await CurveLPMigrator.estimateGas.gas(); 
+    const gas = await CurveLPMigrator.estimateGas.migrate_to_new_pool(); 
```

Comment:
`CurveLPMigrator.estimateGas.migrate_to_new_pool(oldpool, newpool, balace);`  ?
Would you test it locally?  I'm not 100% sure it's correct.

Reply:
Thanks for this. I have updated and believe that is the correct way to estimate gas, however when testing locally I am not able to get the metamask transaction to trigger. I am not sure whats causing this issue.

Reply:
> `CurveLPMigrator.estimateGas.migrate_to_new_pool(oldpool, newpool, balace);` ?
> Would you test it locally? I'm not 100% sure it's correct.

Got it resolved, see latest commit.